src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -100,7 +101,15 @@ public async Task RunAsync()
             StepHost.OutputDataReceived += OnDataReceived;
             StepHost.ErrorDataReceived += OnDataReceived;
 
-            string file = Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Externals), "node", "bin", $"node{IOUtil.ExeExtension}");
+            bool useNode10 = ExecutionContext.Variables.GetBoolean("AGENT_USE_NODE10_FOR_NODE_TASKS") ??
+                StringUtil.ConvertToBoolean(System.Environment.GetEnvironmentVariable("AGENT_USE_NODE10_FOR_NODE_TASKS"), false);
```

Comment:
add `using System;` so you don't have to type `System.`

Reply:
Updated vss library where the `Environment` contract was renamed to `DistributedTaskEnvironment`, so the namespace collision no longer happens.